.circleci/config.yml

```diff
@@ -117,8 +109,24 @@ commands:
 jobs:
   linting_checks:
     <<: *job_defaults
+    executor: python/default
     steps:
-      - lint_setup
+      - checkout:
+          path: ~/curation
+      - python/install-packages:
+          pkg-manager: pip
+          app-dir: ~/curation
+          include-branch-in-cache-key: false
+          include-python-in-cache-key: true
+          pip-dependency-file: data_steward/requirements.txt
+          pypi-cache: true
+      - python/install-packages:
+          pkg-manager: pip
+          app-dir: ~/curation
+          include-branch-in-cache-key: false
+          include-python-in-cache-key: true
+          pip-dependency-file: data_steward/dev_requirements.txt
+          pypi-cache: true
```

Comment:
This seems to be the same as test_setup but without the gcloud sdk. Can that be split up into 'install gcloud sdk' and 'install py packages' so that only the 'install py packages' can be referenced here? test_setup would then reference the two steps under it.

Reply:
it could.  but I'm not interested in optimizing this too much since daniel is planning to rewrite most of this with the dockerized version anyway.  i just want to get our tests working.